chroma-common/blockdevices/blockdevice_zfs.py

```diff
@@ -5,13 +5,13 @@
 import os
 import errno
 import re
-import threading
 import time
 
 from collections import defaultdict
-
+from lockfile import LockFile, LockTimeout
```

Comment:
yes , but given there's not a drop in replacement (I've looked and the replacement behaves differently), it should be follow-on work. I think we should fix the existing bugs to get zfs testing enabled first